library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -405,6 +424,33 @@ public int getSectionPosition(String tag) {
     }
 
     /**
+     * Return the section position in the adapter.
+     *
+     * @param section This section
+     * @return position of the section in the adapter
+     */
+    public int getSectionPosition(Section section) {
```

Comment:
In that new PR, please also change `getSectionPosition(String)` to call `getValidSectionOrThrowException` then this new method.

Then please adapt the unit tests accordingly.